src/packages/documents/document-types/workspace/views/design/document-type-workspace-view-edit.element.ts

```diff
@@ -130,9 +125,9 @@ export class UmbDocumentTypeWorkspaceViewEditElement extends UmbLitElement imple
 				(isSorting) => {
 					this._sortModeActive = isSorting;
 					if (isSorting) {
-						this.#sorter.setModel(this._tabs!);
+						this.#sorter.enable();
```

Comment:
It looks like this new disable/enable feature is not really fully working. I am able to still move things around (groups) with and without going into sort mode.

Additionally, it seems like the UI sometimes breaks as well between sort mode and normal - I think this happens when i start sorting without going into sort mode.